scrapybot/scrapybot/spiders/rusty.py

```diff
@@ -0,0 +1,31 @@
+import re
+import uuid
+from .utils import strip_tags
+from scrapy.linkextractors import LinkExtractor
+from scrapy.spiders import CrawlSpider, Rule
+
+
+class RustySpider(CrawlSpider):
+    name = "rusty"
+    allowed_domains = ["rusty.ozlabs.org"]
```

Comment:
I don't want all of these, some are quite unrelated to bitcoin: https://rusty.ozlabs.org/?paged=11